packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -34,15 +34,25 @@ import {
   useFieldTree,
   useFlattenedFields,
 } from '../../core/components/variableTrees/hooks';
-import { SemiTransparentHeader } from './SemiTransparentHeader';
+import { MapHeader } from './MapHeader';
 import FilterChipList from '../../core/components/FilterChipList';
 import { VariableLinkConfig } from '../../core/components/VariableLink';
 import { MapSideNavigation } from './MapSideNavigation';
-import { SiteInformationProps } from '..';
+import {
+  mapSideNavigationActiveMenuItemBackground,
+  SiteInformationProps,
+} from '..';
 import FloatingVizManagement from './FloatingVizManagement';
 import { InputVariables } from '../../core/components/visualizations/InputVariables';
 import { useToggleStarredVariable } from '../../core/hooks/starredVariables';
 import { filtersFromBoundingBox } from '../../core/utils/visualization';
+import {
+  BarChartSharp,
+  EditLocation,
+  InfoSharp,
+  Save,
+  Share,
+} from '@material-ui/icons';
```

Comment:
**Note to benevolent reviewer(s)**: these were added to flesh out the side navigation as per the discussed on [2023-03-23's Map UX meeting](https://veupathdb.atlassian.net/wiki/spaces/WG/pages/227573761/2023-03-23+Map+UX).

<img width="186" alt="image" src="https://user-images.githubusercontent.com/24446573/228050774-61bf8668-bc01-4bf8-bccb-0f350f605fd8.png">